main.py

```diff
@@ -1,4 +1,12 @@
-from pathlib import Path
+# Discord Version check
+
+import discord
+
+if discord.version_info.major < 1:
+    print("You are not running the rewritten version of discord.py.\n\n"
+          "In order to use Red v3 you MUST be running d.py version"
+          " >= 1.0.0.")
+    sys.exit(1)
```

Comment:
Either:
- `exit(1)` instead of `sys.exit(1)`
OR
- make sure to `import sys` before trying to call `sys.exit`